yb-voyager/src/pgss/pgss_integration_test.go

```diff
@@ -0,0 +1,119 @@
+//go:build integration
+
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package pgss
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"strings"
+	"testing"
+
+	_ "github.com/jackc/pgx/v5/stdlib"
+	testcontainers "github.com/yugabyte/yb-voyager/yb-voyager/test/containers"
+)
+
+// TODO: Keep this in sync with yb-voyager's supported PostgreSQL versions
+var supportedPgVersions = []string{"11", "12", "13", "14", "15", "16", "17"}
+
+func TestPgStatStatementsRequiredColumns(t *testing.T) {
```

Comment:
what is the purpose of this test? 
seems more like we're validating what pg_stat_statements returns on each pg version. Why would we need to continually run this?